live-objects/src/test/kotlin/io/ably/lib/objects/unit/LiveObjectTest.kt

```diff
@@ -0,0 +1,15 @@
+package io.ably.lib.objects.unit
+
+import io.ably.lib.objects.unit.setup.UnitTest
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+import kotlin.test.assertNotNull
+
+class LiveObjectTest : UnitTest() {
```

Comment:
Don't like inheritance in the tests, could you take a look at [this SO answer](https://stackoverflow.com/questions/27391055/how-to-share-junit-beforeclass-logic-among-multiple-test-classes) I think we can setup common parts the same way